tests/GlossaryControllerTest.php

```diff
@@ -2,21 +2,38 @@
 
 namespace TheSceneman\SilverStripeGlossary\Tests;
 
-use SilverStripe\Control\HTTPRequest;
 use SilverStripe\Dev\FunctionalTest;
-use TheSceneman\SilverStripeGlossary\Controllers\GlossaryController;
+use SilverStripe\Security\Member;
 
 class GlossaryControllerTest extends FunctionalTest
 {
 
     /**
-     * Tests if API call gives status code 200 (OK response)
+     * @inheritDoc
      */
-    public function testGlossary()
+    protected static $fixture_file = '/tests/fixtures/Member.yml';
+
+    /**
+     * Tests that the API returns a 403 status code when the user is not logged in
+     */
+    public function testGlossaryNotLoggedIn(): void
+    {
+        $page = $this->get('api/glossary/');
```

Comment:
In order to get these tests working I needed to change the endpoint to `glossary-api/glossary` to match the routes in `glossary.yml`. If I remember correctly I changed the routing along the way as `api` was too generic, and I obviously didnt update the existing test. Would you be able to please update these two tests to use the correct endpoint?

Reply:
Ah good spotting. I've fixed these up :)